mappings/base_mapping.py

```diff
@@ -29,6 +29,22 @@ def applyMapping(self):
 
     def applyFormatting(self):
         return self.record
+    
+    def format_identifiers(self, identifiers, source):
+        identifier_type_dict = {'8': 'issn', '10': 'isbn', '13': 'isbn'}
```

Comment:
I think you can just use the numbers as the keys here so you don't need to use the str() function below. Unless there is a best practice against using nums as keys that I am unaware of.

Reply:
For some reason I recall a time when I put an integer as a key and I got a KeyError but from looking it up and testing it with this code I see that integers can serve as keys for python dictionaries. I appreciate the observation.